docs/data/product/dea-surface-reflectance-landsat-5-tm/_data.yaml

```
@@ -15,8 +15,10 @@ spatial_data_type: Raster
 time_span:
   start: 16/08/1986
   end: 17/11/2011
-update_frequency: Daily
-next_update: null
+data_updates:
+  frequency: DAILY
+  activity: ONGOING
```

Landsat 5 no longer collecting data
```suggestion
  activity: NO_UPDATES
```

---

Ditto the above: NO_UPDATES vs DECOMMISSIONED